draft-ietf-quic-ack-frequency.md

```diff
@@ -633,6 +623,15 @@ Therefore, an endpoint that has sent an ACK_FREQUENCY frame earlier in the
 connection ought to send a new ACK_FREQUENCY frame upon confirmation of
 connection migration with updated information, e.g. to consider the new RTT estimate.
 
+# Setting the Reordering Threshold Value {#set-threshold}
+
+To ensure timely loss detection, a data sender can send a Reordering Threshold
+value that is the same as the loss detection packet threshold. If the
+reordering threshold is smaller than the packet threshold, an acknowledgement is
+unnecessarily sent before the packet can be declared lost. If the value is
+larger, it can cause unnecessary delays in loss detection
+({{Section 6.1.1 of QUIC-RECOVERY}}).
+
```

Comment:
```suggestion

In order to avoid unnecessary immediate acknowledgements, senders SHOULD implement adaptive packet threshold loss detection and communicate the increased Reordering Threshold value to the receiver.

```

@ianswett I've borrowed the text from https://github.com/quicwg/ack-frequency/pull/322#discussion_r2347587598; WDYT?

Reply:
I'd prefer this merge this PR first and then have a separate PR for this addition.